draft-ietf-masque-h3-datagram.md

```diff
@@ -877,11 +497,7 @@ STREAM(44): HEADERS             -------->
   :scheme = https
   :path = /target.example.org/443/
   :authority = proxy.example.org:443
-
-STREAM(44): DATA                -------->
-  Capsule Type = REGISTER_DATAGRAM
-  Datagram Format Type = UDP_PAYLOAD
-  Datagram Format Additional Data = ""
+  sec-capsule-protocol = ?1
```

Comment:
I note the response doesn't have the equivalent header. I assume that's intended but I'm unclear why based on the header's definition.

Reply:
No that was an oversight, fixed.